Web/CSS/Css/Transforms.css

```diff
@@ -0,0 +1,41 @@
+@CHARSET "ISO-8859-1";
+
+/* To define css for body*/
+
+body {
+	background-color: #6699FF;
+	padding-left: 300px;
+}
+
+/* To define css for all div.*/
+
+body div {
```

Comment:
Use class name for the div to this css is to be applied. Otherwise this will applied all divs under body